src/lib/libcmd/cmdext.h

```diff
@@ -72,7 +72,7 @@ extern  int	b_mkdir (int, char**, Shbltin_t*);
 //extern  int	b_sha512sum (int, char**, Shbltin_t*);
 //extern  int	b_stty (int, char**, Shbltin_t*);
 //extern  int	b_sum (int, char**, Shbltin_t*);
-//extern  int	b_sync (int, char**, Shbltin_t*);
+extern  int	b_sync (int, char**, Shbltin_t*);
```

Comment:
I'd really prefer we not include this one at this time. The only way this can be justified is if ksh is meant to be an alternative to something like [busybox](https://busybox.net/about.html). Which might be a reasonable objective but I think it needs to be discussed. The `sync` command is unlike every other builtin.

Reply:
I have mixed feelings about it. It was built by default in the last stable and the implementation looks trivial, so it should not be too much of a burden to maintain. I think we should ship it in the next release.

Reply:
> It was built by default in the last stable.

I confirmed it's in ksh93u+ as shipped with macOS. So, okay, include it. But this is another example of the lack of clear thinking (such as including math functions like tgamma) leading to including a feature seemingly for no reason other than "we can so why not".